testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/deploy/runtime/jaxrs/HelloApplication.java

```diff
@@ -38,6 +38,8 @@
         final Set<Class<?>> classes = new HashSet<>();
         classes.add(HelloResource.class);
         classes.add(PureProxyEndPoint.class);
+        classes.add(HelloBadResource.class);
```

Comment:
This change causes regression of JaxrsRuntimeNameTestCase. JaxrsRuntimeNameTestCase uses HelloApplication.java as well, but HelloBadResource is not added to the deployment from JaxrsRuntimeNameTestCase. So JaxrsRuntimeNameTestCase fails.